e2etest/l2tests/assignment.go

```diff
@@ -148,16 +148,17 @@ var _ = ginkgo.Describe("IP Assignment", func() {
 					AllocateTo: &metallbv1beta1.ServiceAllocation{Priority: 10, Namespaces: []string{f.Namespace.Name}},
 				},
 			}
-			defaultPool := metallbv1beta1.IPAddressPool{
+			defaultNsPool := metallbv1beta1.IPAddressPool{
```

Comment:
this should be "withNamespaceNoPriority", according to the other tests. Also, if we pin here we can revert the pool to be /32